types/node/process.d.ts

```diff
@@ -295,6 +295,8 @@ declare module "process" {
 
                 resourceUsage(): ResourceUsage;
 
+                traceDeprecation: boolean;
```

Comment:
```suggestion
                readonly traceDeprecation?: true;
```
It seems this is either set to `true` or undefined.

Reply:
One should be able to set it to true, so it shouldn't be readonly.

Reply:
Try it in repl - you can't set it to true. It's true only if command line argument `--trace-deprecation` is set.

Reply:
I just tried the following code in Node (v15.0.1 and v10.19.0)

```js
const myWarning = new Error('Something happened')
myWarning.name = 'DeprecationWarning'
process.emitWarning(myWarning)
process.traceDeprecation = true
process.emitWarning(myWarning)
```

The second call to `emitWarning` prints the stack trace so I believe it can be set to true.

Reply:
To be precise I used `docker run --rm -it node:latest node` to have a clean environment.

Reply:
Seems you are right, if `--trace-deprecation` is not on command line it's possible to set it. If it is on command line it readonly.